server/socketHandlers/getPlayers.js

```diff
@@ -0,0 +1,22 @@
+import { formatOutput } from '../formatters/socketHandlerFormatter';
+
+const EVENT_TYPES = {
+  ON: 'getPlayers',
+  EMIT: 'resultGetPlayers'
+}
+
+const getPlayers = (socket, socketNameSpace) => {
```

Comment:
I think the name of this function  should be `getPlayerIds` because `getPlayers` sound like this method returns list of users like below. But actually this function returns list of user id.

```js
[
  {id:1, name:'taro1', location: {}, ...},
  {id:2, name:'taro2', location: {}, ...},
  {id:3, name:'taro3', location: {}, ...},
  ...
]
```